extensions/Cake.ExcelDnaPack.yml

```diff
@@ -0,0 +1,23 @@
+Type: Addin
+Name: Cake.ExcelDnaPack
+NuGet: Cake.ExcelDnaPack
+Assemblies:
+- "/**/Cake.ExcelDnaPack.dll"
+Repository: https://github.com/augustoproiete/Cake.ExcelDnaPack.git
+ProjectUrl: https://github.com/augustoproiete/Cake.ExcelDnaPack/
+Author: augustoproiete
+Description: Cake add-in that makes ExcelDnaPack available as a tool in Cake.
+Categories:
+- exceldnapack
+- exceldna
+- cake-addin
+- cake-build
+- augustoproiete
```

Comment:
Categories need manually to be cleaned up: `cake-addin`, `cake-build`, `augustoproiete`